packages/flutter_tools/lib/src/commands/build_ios.dart

```diff
@@ -103,6 +108,44 @@ class BuildIOSCommand extends BuildSubCommand {
       throwToolExit('Encountered error while building for $logTarget.');
     }
 
+    if (buildInfo.codeSizeDirectory != null) {
+      final SizeAnalyzer sizeAnalyzer = SizeAnalyzer(
+        fileSystem: globals.fs,
+        logger: globals.logger,
+        processUtils: processUtils,
+        appFilenamePattern: 'App'
```

Comment:
Maybe not the case but I somewhat expected this to be a configurable variable somewhere